dev-util/sccache/sccache-0.3.0.ebuild

```diff
@@ -0,0 +1,394 @@
+security-framework-sys-2.6.1
+semver-0.9.0
+semver-parser-0.7.0
+serde-1.0.136
+serde_derive-1.0.136
+serde_json-1.0.79
+serde_qs-0.8.5
+serde_repr-0.1.7
+serde_urlencoded-0.7.1
+serial_test-0.5.1
+serial_test_derive-0.5.1
+sha-1-0.10.0
+sha1-0.6.1
+sha1_smol-1.0.0
+sha2-0.10.1
+signal-hook-registry-1.4.0
+simple_asn1-0.4.1
+skeptic-0.4.0
+slab-0.4.5
+smallvec-1.8.0
+socket2-0.4.4
+spin-0.5.2
+static_assertions-1.1.0
+stringmatch-0.3.3
+strip-ansi-escapes-0.1.1
+strsim-0.8.0
+subtle-2.4.1
+syn-1.0.86
+syslog-5.0.0
+tar-0.4.38
+tempdir-0.3.7
+tempfile-3.3.0
+termcolor-1.1.2
+termtree-0.2.4
+textwrap-0.11.0
+thirtyfour-0.27.3
+thirtyfour_sync-0.27.1
+thiserror-1.0.30
+thiserror-impl-1.0.30
+threadpool-1.8.1
+time-0.1.43
+time-0.3.2
+tiny_http-0.8.2
+tinyvec-1.5.1
+tinyvec_macros-0.1.0
+tokio-1.16.1
+tokio-macros-1.7.0
+tokio-native-tls-0.3.0
+tokio-serde-0.8.0
+tokio-util-0.6.9
+toml-0.5.8
+tower-0.4.11
+tower-layer-0.3.1
+tower-service-0.3.1
+tracing-0.1.30
+tracing-attributes-0.1.19
+tracing-core-0.1.22
+try-lock-0.2.3
+twoway-0.1.8
+typenum-1.15.0
+unicase-2.6.0
+unicode-bidi-0.3.7
+unicode-normalization-0.1.19
+unicode-width-0.1.9
+unicode-xid-0.2.2
+unix_socket-0.5.0
+untrusted-0.7.1
+url-2.2.2
+urlparse-0.7.3
+utf8parse-0.2.0
+uuid-0.8.2
+vcpkg-0.2.15
+vec_map-0.8.2
+version-compare-0.1.0
+version_check-0.9.4
+void-1.0.2
+vte-0.10.1
+vte_generate_state_changes-0.1.1
+wait-timeout-0.2.0
+waker-fn-1.1.0
+walkdir-2.3.2
+want-0.3.0
+wasi-0.9.0+wasi-snapshot-preview1
+wasi-0.10.2+wasi-snapshot-preview1
+wasm-bindgen-0.2.79
+wasm-bindgen-backend-0.2.79
+wasm-bindgen-futures-0.4.29
+wasm-bindgen-macro-0.2.79
+wasm-bindgen-macro-support-0.2.79
+wasm-bindgen-shared-0.2.79
+web-sys-0.3.56
+which-4.2.4
+winapi-0.2.8
+winapi-0.3.9
+winapi-build-0.1.1
+winapi-i686-pc-windows-gnu-0.4.0
+winapi-util-0.1.5
+winapi-x86_64-pc-windows-gnu-0.4.0
+winreg-0.7.0
+wiremock-0.4.9
+xattr-0.2.2
+zip-0.5.13
+zstd-0.6.1+zstd.1.4.9
+zstd-safe-3.0.1+zstd.1.4.9
+zstd-sys-1.4.20+zstd.1.4.9
+"
+
+inherit cargo optfeature
+
+DESCRIPTION="ccache/distcc like tool with support for rust and cloud storage"
+HOMEPAGE="https://github.com/mozilla/sccache/"
+
+if [ ${PV} == "9999" ] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/mozilla/sccache.git"
+else
+	SRC_URI="https://github.com/mozilla/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	$(cargo_crate_uris ${CRATES})"
+	KEYWORDS="~amd64 ~ppc64"
+fi
+
+LICENSE="Apache-2.0 Apache-2.0-with-LLVM-exceptions BSD BSD-2 Boost-1.0 ISC MIT Unlicense ZLIB"
+SLOT="0"
+IUSE="azure dist-client dist-server gcs memcached redis s3"
+
+BDEPEND="virtual/pkgconfig"
+
+DEPEND="
+	app-arch/zstd
+	dist-server? ( dev-libs/openssl:0= )
+	gcs? ( dev-libs/openssl:0= )
+"
+
+RDEPEND="${DEPEND}
+	dist-server? ( sys-apps/bubblewrap )
+"
+
+QA_FLAGS_IGNORED="usr/bin/sccache*"
+
+src_unpack() {
+	if [[ "${PV}" == *9999* ]]; then
+		git-r3_src_unpack
+		cargo_live_src_unpack
+	else
+		cargo_src_unpack
+	fi
+}
+
+src_configure() {
+	myfeatures=(
```

Comment:
add `native-zlib` here, unconditionally.

Reply:
and ofc don't forget to add a dep

Reply:
I added `native-zlib`, thanks for the review!